File: Katiba55.API/Entities/Item.cs
namespace Katiba55.API.Entities
{
    public class Item: BaseEntity
    {
        public string Name { get; set; }
        public DateTime? EstimatedStartDate { get; set; }
        public DateTime? EstimatedEndDate { get; set; }
        public DateTime? ActualStartDate { get; set; }
        public DateTime? ActualEndDate { get; set; }
        public double? TotalValue { get; set; }
        public double? ExecutedValue { get; set; }
        public double? RemainingValue { get; set; }
        public double? RelativeWeightPercent { get; set; }
        public double? ExecutionPercent { get; set; }
        public DateTime? ExecutionDate { get; set; }
        public ExecutionStatus ExecutionStatus { get; set; }
        public string? Notes { get; set; }

        public int WorkId { get; set; }
        public Work Work { get; set; }
    }
}

File: Katiba55.Client/src/app/models/items/item.ts
export interface Item {
    id: number;
    name: string;
    estimatedStartDate?: Date;
    estimatedEndDate?: Date;
    actualStartDate?: Date;
    actualEndDate?: Date;
    totalValue?: number;
    executedValue?: number;
    remainingValue?: number;
    relativeWeightPercent?: number;
    executionPercent?: number;
    executionDate?: Date;
    executionStatus: string;
    notes?: string;
}
